XmlFormatter/src/Windows/PluginManager.cs

```diff
@@ -0,0 +1,245 @@
+﻿using PluginFramework.src.DataContainer;
+using PluginFramework.src.Interfaces.Manager;
+using PluginFramework.src.Interfaces.PluginTypes;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Windows.Forms;
+using XmlFormatter.src.Interfaces.Settings;
+using XmlFormatter.src.Interfaces.Settings.DataStructure;
+using XmlFormatter.src.Settings.DataStructure;
+
+namespace XmlFormatter.src.Windows
+{
+    /// <summary>
+    /// Window to manage the plugins
+    /// </summary>
+    public partial class PluginManager : Form
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5528129)